### ISSUE TITLE
Sass 3.3.2 index and maps does not appear to work as per the documentation

### ISSUE THREAD

**Issue by mikehdt** [sass/sass#1181]
I've been exploring the new maps in Sass 3.3, and seem to have encountered an issue combining them with index as per the documentation's example.

Per http://sass-lang.com/documentation/Sass/Script/Functions.html#index-instance_method I have made this SCSS:

``` scss
.test {
    exists: index((width: 10px, height: 20px), (height, 20px));
}
```

According to the documentation, this should return `2`. However, CodeKit 2.0.1 (running Sass 3.3.2) is outputting `false` instead. Am I doing something wrong, or is this a bug?

Also, it's a little unclear from the documentation, but can `index()` find with just a key, not necessarily matching against a key AND value together for `$value`?

**Comment by nex3** [comment]
This is expected; Sass considers comma-separated lists to be different than space-separated lists, and it considers maps to be equivalent to comma-separated lists of space-separated elements.

The real issue here is that the documentation is incorrect. I'll fix that.

> Also, it's a little unclear from the documentation, but can `index()` find with just a key, not necessarily matching against a key AND value together for `$value`?

No; you should use `map-get` for that.

**Comment by nex3** [comment]
Fixed by c7e8acfe5e3fdf95fa427b5227a45aa9f863a2ea.
